src/mediapipe_internal/mediapipegraphdefinition.cpp

```diff
@@ -380,6 +390,15 @@ std::string MediapipeGraphDefinition::getStreamName(const std::string& streamFul
     return EMPTY_STREAM_NAME;
 }
 
+PythonNodeResource* MediapipeGraphDefinition::getPythonNodeResource(const std::string& nodeName) {
```

Comment:
There is really no point in using std::shared_ptr if you access the objects via raw pointers anyway.
Either keep on passing around shared pointers (which would be slower?)
Or keep using raw pointers and keep resources without stl unique/shared pointer